cfpb/consumerfinance.gov
# P4C: Show community-college-specific text

Displays "2 year" instead of "4 year" in text around gas gauge charts when selected school's highest degree is associate.




## Additions

- New classes to toggle "2 year" and "4 year" text



## Changes

- Update a few external links to open in new tab


## How to test this PR

1. Check out branch and run gulp
2. Navigate to [grad path tool with community college selected](http://localhost:8000/paying-for-college/your-financial-path-to-graduation/?iped=174428&houp=offCampus&typp=associates&lenp=2&ratp=inState&depp=independent&cobs=n&tuit=5791&hous=8260&book=1000&pers=15051&houx=1013&fdx=417&clhx=86&trnx=569&hltx=299&entx=151&retx=196&taxx=77&othx=258), go to "Compare this school to others" tab, and verify that text around gas gauges references "2 year" instead of "4 year" colleges
e. Navigate to [grad path tool with 4 year college selected](http://localhost:8000/paying-for-college/your-financial-path-to-graduation/?iped=106397&pid=5203-3&houp=onCampus&typp=bachelors&lenp=4&ratp=inState&depp=dependent&houx=1037&fdx=418&clhx=100&trnx=550&hltx=265&entx=140&retx=192&taxx=23&othx=252), go to "Compare this school to others" tab, and verify that text around gas gauges references "4 year" colleges

## Screenshots
### Community college

<img width="699" alt="Screen Shot 2021-02-10 at 11 42 06 AM" src="https://user-images.githubusercontent.com/778171/107563923-84127280-6b96-11eb-927e-68fecde3bdad.png">


### 4 year college


<img width="690" alt="Screen Shot 2021-02-10 at 11 42 42 AM" src="https://user-images.githubusercontent.com/778171/107563928-85dc3600-6b96-11eb-93e3-317e67d89b03.png">



## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
